apps/web/src/utils/encodePageNumToEndCursor.test.ts

```diff
@@ -5,18 +5,18 @@ import { describe } from 'vitest'
 import { encodePageNumToEndCursor } from './encodePageNumToEndCursor'
 
 const decode = (cursor: string) => {
-  let decoded = ''
+  let decoded = '';
   for (const char of base64.decode(cursor)) {
-    decoded += String.fromCharCode(char)
+    decoded += String.fromCharCode(char);
   }
-  return decoded
+  return decoded;
 }
 
 describe('Encode Page Number to End Cursor String', () => {
   it('should, given no page parameter and a limit of 20, return a skip of 0', () => {
     const result = encodePageNumToEndCursor(20, '')
-    let decoded = decode(result)
-    assert.equal(decoded, '{"skip":0}')
+    let decoded = decode(result);
+    assert.equal(decoded, '{"skip":0}');
```

Comment:
why is the diff adding semi colons? we don't use semi colons 🤔

Reply:
I was adding semi colons to test the hook but it wasn't being removed pre-push -- trying to resolve this today -- attempting to add `git add` to the .husky prepush file doesn't actually solve this problem, so was attempting to add a pre-commit hook

Reply:
ahh gotcha ok thanks :)